framework/utils/xmlUtils.py

```diff
@@ -206,16 +206,16 @@ def fixXmlText(msg):
     @ Out, msg, string, fixed string
   """
   #if not a string, pass it back through
-  if not isinstance(msg,basestring):
+  if not type(msg).__name__ in ['unicode','str']:
```

Comment:
The issue seems to be that adding the `framework/utils` directory to the path does not let python make itself aware that the `utils` is a module, thus directly importing `xmlUtils` makes it so that it is not aware of siblings within its own parent module, I guess?  By backing out one directory, python can be made aware that there is a larger module called `utils`, and can find `.utils` beside `xmlUtils` in the module.